docs/en/engines/table-engines/special/distributed.md

```diff
@@ -25,10 +25,19 @@ The Distributed engine accepts parameters:
     -   [insert_distributed_sync](../../../operations/settings/settings.md#insert_distributed_sync) setting
     -   [MergeTree](../../../engines/table-engines/mergetree-family/mergetree.md#table_engine-mergetree-multiple-volumes) for the examples
 
+Also it accept the following settings:
+
+- `fsync_after_insert` - Do fsync for every inserted. Will decreases performance of inserts (only for async INSERT, i.e. `insert_distributed_sync=false`),
+
+- `fsync_tmp_directory` - Do fsync for temporary directory (that is used for async INSERT only) after all part operations (writes, renames, etc.).
```

Comment:
Difference of those 2 setting is not very clear. Better description highlighting the difference is needed.

Reply:
An suggestion?

Reply:
I'm wondering if there is any use case when one fsync should be enables while the other - not. 

I guess the people who want guarantees and ok to make inserts slower, don't expect to get half-guarantee.

Reply:
>I'm wondering if there is any use case when one fsync should be enabled while the other - not.

Most likely people tend to enable fsync but not fsync for directories
Also it varies from the underlying filesystem.

Reply:
> Most likely people tend to enable fsync but not fsync for directories

Questionable, but OK. 

> An suggestion?

See the comment below.